travis/getaddons.py

```diff
@@ -71,7 +71,7 @@ def get_addons(path):
         res = [path]
     else:
         res = [os.path.join(path, x)
-               for x in os.listdir(path)
+               for x in sorted(os.listdir(path))
```

Comment:
Let me ask a question: Are you insinuating that enterprise dependencies would get installed into the `odoo-9.0` folder? That would just break (because of the web module (we can't predict order reliably)), but this is then a problem of `oca_dependency` and unrelated here. But be assured, here is your answer: https://github.com/OCA/maintainer-quality-tools/blob/master/travis/test_server.py#L132-L134 `travis_dependencies_dir` comes first.
Again, within that folder, we cannot predict any order, so this commit is completely `pareto efficient`. Can you explain at least why do you think enterprise needs to come before other `travis_dependencies_dir` - addons. Yes, I'm aware that it needs to come before `odoo-cc` and why.